examples/wifi/main.py

```diff
@@ -106,8 +106,12 @@ def show_wifi_scans(self):
         for name in wifi_scans:
             content = ""
             items = wifi._get_network_info(name)
-            for key, value in items.iteritems():
-                content += "{}:    {} \n".format(key, value)
+            if sys.version_info.major is 3: #Support for Py2 and Py3
+                for key, value in items.items():
+                    content += "{}:    {} \n".format(key, value)
+            else:
+                for key, value in items.iteritems():
```

Comment:
I would just use `items.items()` to keep things simple, it seems unlikely that the dict will have enough keys for it to matter in Python 2.

Reply:
I think I tried using only items.items(). But if someone have time to clean my mess I'd be more than happy!